### DISCUSSION TITLE
Camera Bar Removal...

### DISCUSSION THREAD

**Discussion by w951wcl** [thumbsup/thumbsup#245] [General]
Can the Camera Name bar be removed?  Some pictures don't have that metadata.
 Thanks

**Comment by rprieto** [comment]
Hi, can you please post a screenshot to illustrate?

**Comment by w951wcl** [comment]
![image](https://user-images.githubusercontent.com/80331476/111007888-7b5bba80-8355-11eb-950e-076c8b9748c1.png)

**Reply by w951wcl** [reply]
On some of the pictures it has the name of the camera in the gray bar, In on my cell phone pictures, it does not even show up.

**Comment by w951wcl** [comment]
![image](https://user-images.githubusercontent.com/80331476/111008176-1f456600-8356-11eb-8c35-b8c424abfa2a.png)

**Reply by rprieto** [reply]
This is strange, it looks like the caption bar displayed in the wrong place. 

Do you see this bar on the official demos too? https://thumbsup.github.io/demos/themes/mosaic/

**Comment by w951wcl** [comment]
No it does not. 

But it is not on all my pictures.

Also, Chrome 89.0 and IE Edge shows it.

IE 20H2 does not.
It also goes away when I hit F12 for debug.

 Theme is Classic

**Comment by w951wcl** [comment]
Hi:

 

I added this and I still see the captions

--show-captions false --default-captions false

 

Thanks for your help!

**Comment by w951wcl** [comment]
I am going to try to attach a example photo that it is happening on.

**Comment by w951wcl** [comment]
I modified the Caption function in Medadata.js to just return null

**Reply by rprieto** [reply]
Hi, yes it looks like the photos have the camera name set as the EXIF or IPTC caption.
I can think of some other options to avoid modifying the source code
- remove the caption using `exiftool`
- clone https://github.com/thumbsup/theme-classic locally, remove the HTML code that shows the caption, and use `--theme-path <yourLocalCopy>`
